manim/cli/project/commands.py

```diff
@@ -0,0 +1,308 @@
+"""Manim's project initialization and management subcommands.
+
+init -  The init subcommand is a quick and easy way to initialize a project
+        It copies files from templates dir and pastes them in the current working dir
+
+new  -  The new command group has 2 commands. these commands handle project creation
+        and scene creation or insertion
+
+        project -   The project subcommand is used for project creation
+                    This command is similar to init but different in a way
+                    that it asks for project_name and template_name.
+                    init command initializes a new project in the current dir
+                    while project command creates a new folder and creates the
+                    project there
+
+        scene   -   The scene subcommand command is used for inserting new scenes.
+                    scene command can create new files and insert new scenes in there.
+                    It can also insert new scenes in an already existing python file.
```

Comment:
I'd greatly prefer if the files structure were made more modular. That is:

- `manim/cli/new/` is its own directory
- `manim/cli/new/commands.py` is its own file that contains the `project`/`scene` decorators
- `manim/cli/init` is its own directory.
- `manim/cli/init/commands.py` is its own file.

I'm okay with the file under `manim/cli/new/` being renamed to `group.py` if you feel that's more appropriate. If that's done, the same should also be done for `manim/cli/cfg`.

Reply:
Okay i can do that. that would be easy to navigate

Reply:
Done